web/components/cycles/quick-actions.tsx

```diff
@@ -78,7 +78,7 @@ export const CycleQuickActions: React.FC<Props> = observer((props) => {
           title: "Restore success",
           message: "Your cycle can be found in project cycles.",
         });
-        router.push(`/${workspaceSlug}/projects/${projectId}/cycles/${cycleId}`);
+        router.push(`/${workspaceSlug}/projects/${projectId}/archives/cycles`);
```

Comment:
Feel like this router.push is unnecessary since the user will have to be in that page to execute this action